src/borg/helpers.py

```diff
@@ -1588,9 +1588,11 @@ def __init__(self, format):
     def get_item_data(self, archive):
```

Comment:
note: archive is now an Archive instance, but used to be only an "archive info" (manifest entry).